yarn.lock

```diff
@@ -21,6 +21,11 @@
   resolved "https://registry.yarnpkg.com/@agoric/assert/-/assert-0.5.2-dev-669ad6d.0.tgz#91ea15ddd228d2209179103dae1337267baa07e8"
   integrity sha512-yUPwL6fqdsdtmVdQqw9s6+2O5HXt317GXdzKrMkzqJyEXNEPsFqFHRrsMJoVZZFje3Xi1r37dPVE6i2QSCwcUg==
 
+"@agoric/assert@0.5.2-dev-caf0bf0.0+caf0bf0":
```

Comment:
are the `yarn.lock` changes necessary? I don't see any package.json changes

Reply:
I did add @agoric/inter-protocol as a dep

Reply:
derp, I didn't see package.json in the list because it was at the top :)